### ISSUE TITLE
CV outputs not 100% accurate

### ISSUE THREAD

**Issue by anemlife** [16n-faderbank/16n#59]
Hello,

First of all, sorry for posting there but I've been reaching Michigan Synth Work for my issue with 0 answers from them (my 16n was built by them).

I use the 16n faderbank mostly for the CV outputs and recently realized these were not going to full 5V. (I have to slightly offset the value with another module to reach a 100% wet value) The problem is: I don't own 16 offsets and it's getting a big issue for me! I already tried other USB cables/power sources thinking it could be the problem, but they all give the same results so this is probably not the issue.

Is there any workaround/solution for this? Thanks!

**Comment by infovore** [comment]
Not really. The fader outputs draw power from your USB +5V line, which is meant to be around +5V but is more likely to be in the 4.90-5.00V region. The only way to avoid this is with external power, filtered down to exactly 5V; the Eurorack-capable revisions, such as the Sweet Sixteen or the AtoV rework will avoid this _as long as they are powered by 12V external_ (which will then be regulated down much more closely to 5V.
